apps/sms/views/shared/test/unit/time_headers_test.js

```diff
@@ -233,43 +241,59 @@ suite('TimeHeaders > ', function() {
   });
 
   suite('TimeHeaders.update', function() {
-    var subject, formattedTime, formattedDate;
+    var subject, time;
 
     setup(function() {
       subject = document.createElement('header');
       subject.dataset.timeUpdate = 'repeat';
-      var time = Date.parse('2013-01-01');
+      time = Date.parse('2013-01-01');
       subject.dataset.time = time;
-      formattedTime = Utils.getFormattedHour(time);
-      formattedDate = Utils.getHeaderDate(time);
     });
 
     test('date and time header', function() {
       TimeHeaders.update(subject);
 
+      var formatter = new Intl.DateTimeFormat(navigator.languges, {
+        month: 'numeric',
+        day: 'numeric',
+        year: 'numeric',
+        hour12: navigator.mozHour12,
+        hour: 'numeric',
+        minute: 'numeric'
+      });
+
       var content = subject.textContent;
-      assert.include(content, formattedTime);
-      assert.include(content, formattedDate);
+      assert.equal(content, formatter.format(time));
     });
 
     test('date header', function() {
       subject.dataset.dateOnly = 'true';
       TimeHeaders.update(subject);
 
+      var formatter = new Intl.DateTimeFormat(navigator.languges, {
+        month: 'numeric',
+        day: 'numeric',
+        year: 'numeric',
+      });
+
       var content = subject.textContent;
-      assert.isTrue(content.indexOf(formattedTime) === -1,
+      assert.equal(content, formatter.format(time),
                     'subject.textContent should not contain formattedTime');
-      assert.include(content, formattedDate);
     });
 
     test('time header', function() {
       subject.dataset.timeOnly = 'true';
       TimeHeaders.update(subject);
 
+      var formatter = new Intl.DateTimeFormat(navigator.languges, {
+        hour12: navigator.mozHour12,
+        hour: 'numeric',
+        minute: 'numeric'
+      });
+
       var content = subject.textContent;
-      assert.include(content, formattedTime);
-      assert.isTrue(content.indexOf(formattedDate) === -1,
-                    'subject.textContent should not contain formattedDate');
+      assert.equal(content, formatter.format(time),
+          'subject.textContent should not contain formattedDate');
```

Comment:
nit: please use the following style:
```javascript
assert.equal(
  content, formater.format(time),
  'subject...'
);
```

How you arrange things between () is not very strict, things should be as logic as possible; but if you need several line for arguments, then please have the closing and ending parens like this.